examples/field_examples/viking_graben_line12/proc/README.md

```diff
@@ -0,0 +1,18 @@
+# Viking Graben Line 12: processing
+
+The processing is done using [Madagascar](https://www.reproducibility.org/wiki/Main_Page) open source software. Thus we assume that Madagascar is present on the system.
+
+The processing is partly based on [Rodrigo Morelatto work](https://github.com/rmorel/Viking).
```

Comment:
Deghosting and geometry are from mentioned repo. 
The original repo doesn' contain some important procedures like Radon filtration to supress multiples. It would be problematic to use original repo as is. Modifying original repo is also pretty big task at least because of name convention of processed files that I use.
I used it as a starting point.